compiler/code-gen/naming.h

```diff
@@ -166,6 +171,7 @@ class FunctionSignatureGenerator {
   bool final_ = false;
   bool pure_virtual_ = false;
   bool inline_ = false;
+  bool definition_ = false;
```

Comment:
Maybe change to `enum`?
Just it's strange a little bit cognitively complex when you call some functions and add the last parameter true or false